docs/html/topics/https-certificates.md

```diff
@@ -20,43 +19,28 @@ variables.
 
 ## Using system certificate stores
 
-```{versionadded} 22.2
-Experimental support, behind `--use-feature=truststore`.
-As with any other CLI option, this can be enabled globally via config or environment variables.
-```
-
-It is possible to use the system trust store, instead of the bundled certifi
-certificates for verifying HTTPS certificates. This approach will typically
-support corporate proxy certificates without additional configuration.
-
-In order to use system trust stores, you need to use Python 3.10 or newer.
-
-  ```{pip-cli}
-  $ python -m pip install SomePackage --use-feature=truststore
-  [...]
-  Successfully installed SomePackage
-  ```
-
-### When to use
+```{versionadded} 24.2
 
-You should try using system trust stores when there is a custom certificate
-chain configured for your system that pip isn't aware of. Typically, this
-situation will manifest with an `SSLCertVerificationError` with the message
-"certificate verify failed: unable to get local issuer certificate":
-
-```{pip-cli}
-$ pip install -U SomePackage
-[...]
-   SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (\_ssl.c:997)'))) - skipping
 ```
 
-This error means that OpenSSL wasn't able to find a trust anchor to verify the
-chain against. Using system trust stores instead of certifi will likely solve
-this issue.
+If Python 3.10 or later is being used then by default
```

Comment:
```suggestion
On Python 3.10 or later, by default
```

Being more concise is probably a good thing. Also using the verb `using` twice in the same sentence is a bit confusing IMO. I would also suggest making a similar change below when discussing Python 3.9 and below. Feel free to reject this suggestion if you disagree.

Reply:
Updated the wording in https://github.com/pypa/pip/pull/11647/commits/29de6e2a380e39af968b5c98087c6b43f10ac143, thanks for this suggestion!